eng/Packages.Data.props

```diff
@@ -137,7 +137,7 @@
     All should have PrivateAssets="All" set so they don't become pacakge dependencies
   -->
   <ItemGroup>
-    <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20210729.3" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20210730.4" PrivateAssets="All" />
```

Comment:
Certainly not blocking, but surprising there's no other change if you made changes to the generator. Is that expected?